### ISSUE TITLE
Option autoHideScrollbar doesn't work

### ISSUE THREAD

**Issue by bensampaio** [malihu/malihu-custom-scrollbar-plugin#357]
If I understood the documentation correctly when `autoHideScrollbar` is set to false the scrollbar should always be visible. However, be it true or false I always get the same result: scrollbar only visible on hover. Am I interpreting incorrectly the purpose of this option, or is this a bug?

Also tried `alwaysShowScrollbar : 1` but nothing changed...

**Comment by malihu** [comment]
Are you using a (special) scrollbar theme like "minimal"? This theme overwrites some options (`autoHideScrollbar` being one of them). You can copy-paste the theme's CSS and create your own copy of "minimal" theme. For info, [see "Creating a new scrollbar theme" in plugin home](http://manos.malihu.gr/jquery-custom-content-scroller/#styling-section)

**Comment by bensampaio** [comment]
Yes, I am using "minimal". Well that is not clear in the documentation... At least I can't find any reference to "minimal" there. It would have saved me several hours of frustration... Anyway I don't think a theme should overwrite / ignore some of the settings... Thanks anyway!

**Comment by malihu** [comment]
You're right, I'll update the documentation. 

This theme is the only one overwriting this option. Other options overwritten by other themes are mostly by design (e.g. the round circle "rounded" themes are fixed size so they must overwrite the `autoDraggerLength` option). 

I'll consider adding a condition on non-essential options like `autoHideScrollbar` on "minimal" themes (although the themes are in general ready-to-use with certain options set). 

Anyway, it's my bad for not stating it on the documentation so I do apologize :)
